flask_resty/authorization.py

```diff
@@ -5,29 +5,80 @@
 
 
 class AuthorizationBase(object):
+    """Base class for the API authorization scheme. Typically you will want
+    to use the :py:class:`AuthorizeModifyMixin` to implement uniform
+    authorization logic for each CRUD operation.
```

Comment:
"for each CRUD operation" is slightly inaccurate, since it's only for mutation actions (CUD :D)